src/wasm-interpreter.h

```diff
@@ -3136,7 +3140,7 @@ class ModuleRunnerBase : public ExpressionRunner<SubType> {
       return target;
     }
 
-    Index index = target.getSingleValue().geti32();
+    Index index = target.getSingleValue().getInteger();
```

Comment:
The truncation here from assigning to a 32-bit index means that extremely large indices will incorrectly wrapped to be in-bounds. We should use uint64_t or Address until we can do a bounds check.

Reply:
Good point. I'll make this `getUnsigned()` and add a TODO to fix up the usage of this (tableLoad's signature will need changes).